.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   extends: ['@espressive', 'prettier'], // prettier must be last so we override any rules that potentially conflict with Prettier
   overrides: [
     {
-      files: ['**/next-*/**/*', 'docs/**/*'],
+      files: ['**/next-*/**`/*', 'docs/**/*'],
```

Comment:
@lordpixel why was this ` added?

Reply:
Upon looking at the rest of your fixes... the reason you had to add all the React imports to the Next docs is because your ` here broke the lint rules being disabled for React being in JSX scope. React does not need to be in scope in Next.js. You probably had to fight with the anchor valid rule too because of how Next.js does their links.

This ` needs to be removed on line 7 so these rules go back to being disabled. This also means you will want to go pull all the React imports you added to the Next.js files that are not needed.

Reply:
Thanks for catching this.

Reply:
@lordpixel I am still seeing the character on line 7 that needs to be removed. Until that character is removed, there are lint rules are not correctly disabled on Next.js projects
```
'jsx-a11y/anchor-is-valid': 0,
'react/react-in-jsx-scope': 0,
```